### DISCUSSION TITLE
A place to update any changes in libraries used ?!

### DISCUSSION THREAD

**Discussion by dew10120** [Cyfrin/foundry-full-course-cu#1809] [Ideas]
@PatrickAlphaC 
June 8, 2024
Hi Patrick,

There are changes and updates in libraries used. for example, 

openzeppelin/contracts/security/ReentrancyGuard.sol had been moved to 
openzeppelin/contracts/utils/ReentrancyGuard.sol

this is recorded in openzeppelin changelog as below 
- `ReentrancyGuard`, `Pausable`: Moved to `utils` directory. ([#4551](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4551))

It might be good for new learners if anyone that might come across such changes could highlight these changes in a single file or locations, a document similar to changelog.md 

Maybe you or  others have better ideas ??

Thank you

DEW

**Comment by EngrPips** [comment]
Hello @dew10120, Did you check the `chronological update` section of the GitHub resources associated with the course to confirm if those updates haven't been mentioned and accounted for?

**Reply by dew10120** [reply]
Hi, Yes,  I believe this particular change is not updated in the `chronological update` section.

But it can be done !

**Comment by dew10120** [comment]
Hi, @EngrPips are you able to add same to `chronological update` section ? 
tkx

**Reply by EngrPips** [reply]
I would contact the team and have them review and make updates if needed. Thanks for pointing this out.

**Reply by PatrickAlphaC** [reply]
We are doing an overhaul of the codebase and videos. Stay tuned.

**Reply by EngrPips** [reply]
> We are doing an overhaul of the codebase and videos. Stay tuned.

Cant wait hahaha.
